seed/views/properties.py

```diff
@@ -814,7 +816,7 @@ def columns(self, request):
               paramType: query
         """
         organization_id = int(request.query_params.get('organization_id'))
-        only_used = request.query_params.get('only_used', False)
+        only_used = json.loads(request.query_params.get('only_used', 'false'))
```

Comment:
Before, sending `"false"` to this endpoint actually ended up making `only_used` be a truthy, non-empty string. Note, this fix happened for the corresponding endpoint in v3, so no changes needed there.